ckanext/datagov_inventory/tests/logic/auth/test_auth.py

```diff
@@ -378,11 +378,11 @@ def test_auth_user_list(self):
         self.setup_test_orgs_users()
 
         self.assert_user_authorization('user_list', {
-            'gsa_admin': is_denied,
-            'gsa_editor': is_denied,
-            'gsa_member': is_denied,
-            'doi_admin': is_denied,
-            'doi_member': is_denied,
+            'gsa_admin': is_allowed,
```

Comment:
why did these change?

Reply:
No one actually knows. I remember when the tests were written, it was more like describing what is happening, not testing what should be happening.

Reply:
Checked on inventory with ckan 2.10.1 and ckan 2.10.4. The new test result matches the actual behavior on both. So maybe previous test was wrong in 2.10.1 and it got fixed in the new version2.10.4.